src/lib/types/settings.type.ts

```
@@ -0,0 +1,46 @@
+	autoUpdate: boolean;
+	autoUpdateInterval: number;
+	pollingEnabled: boolean;
+	pollingInterval: number;
+	stacksDirectory: string;
+	pruneMode: 'all' | 'dangling';
+	registryCredentials?: Array<{
+		url: string;
+		username: string;
+		password: string;
+	}>;
+	externalServices?: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)